conduit/data/block_export_data.go

```diff
@@ -16,6 +18,8 @@ type InitProvider interface {
 	GetGenesis() *sdk.Genesis
 	SetGenesis(*sdk.Genesis)
 	NextDBRound() sdk.Round
+	GetTelemetryClient() *telemetry.OpenSearchClient
+	SetTelemetryClient(*telemetry.OpenSearchClient)
```

Comment:
I think `Set` can be a property of the constructor / implementation rather than included as part of the interface.

Reply:
Good point, removed from interface in b4acb9aa974320d5743aab8c7bd68eb30c1e9b44